miniDAS/format.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import asdict, dataclass, field
+from pathlib import Path
+from typing import Any, Literal
+
+import h5py
+import numpy as np
+
+DataUnit = Literal["m/m", "cm/m", "mm/m"]
+ScaledUnit = Literal["ue/m"]
+_ALLOWED_DTYPES = (np.int32, np.float32)
```

Comment:
ASN requested to also allow int16
https://github.com/DAS-RCN/RCN_DASformat/issues/7